frontend/public/components/utils/dropdown.jsx

```diff
@@ -292,8 +292,7 @@ export class Dropdown extends DropdownMixin {
     const button = noButton
       ? <div onClick={this.toggle} className="dropdown__not-btn" id={this.props.id}>
         <span className="dropdown__not-btn__title">{title}</span>&nbsp;<Caret />
-      </div>
-      : <button onClick={this.toggle} type="button" className="btn btn-default btn--dropdown dropdown-toggle" id={this.props.id}>
+      </div> : <button onClick={this.toggle} type="button" className={`btn btn-default btn--dropdown dropdown-toggle ${className}`} id={this.props.id}>
```

Comment:
Ideally the colon and everything after is on its own line so the formatting is consistent across the code and the only resulting diff is on line 296.
```
</div> 
: <button onClick={this.toggle} type="button" className={`btn btn-default btn--dropdown dropdown-toggle ${className}`} id={this.props.id}>
```